### PR TITLE
Add support for ex-data and exception message

### PR DESCRIPTION
**Pull request by apmaros**
Implements support for ex-data and exception message.
Resolves issue https://github.com/FundingCircle/clj-loga/issues/21

### CODE REVIEW

File: project.clj [hunk 1]
```diff
@@ -1,4 +1,4 @@
-(defproject clj-loga "0.5.0"
+(defproject clj-loga "0.5.1"
```
Comment (whamilton42) [inline project.clj]: > [MINOR version when you add functionality in a backwards-compatible manner](http://semver.org/)

Comment (apmaros) [inline project.clj]: I consider the PR as a patch which serves for bugfix and is backwards compatible. The functionality was supposed to be there at the first place.

**Comment by whamilton42** [comment]
Nice work.

**Comment by apmaros** [comment]
Thanks @whamilton42, also for pointing out!

**Pull request merged**